homeassistant/components/number/strings.json

```diff
@@ -37,6 +37,9 @@
     "aqi": {
       "name": "[%key:component::sensor::entity_component::aqi::name%]"
     },
+    "area": {
+      "name": "[%key:component::sensor::entity_component::area::name%]"
+    },
```

Comment:
Are we not missing a corresponding `number` icon?

Reply:
Can we decide on what icon to use, my question about this is still unanswered.